init.go

```diff
@@ -50,6 +52,10 @@ PfZ+G6Z6h7mjem0Y+iWlkYcV4PIWL1iwBi8saCbGS5jN2p8M+X+Q7UNKEkROb3N6
 KOqkqm57TH2H3eDJAkSnh6/DNFu0Qg==
 -----END CERTIFICATE-----`
 )
+const (
+	syncToChainDefaultPollingInterval = 3 * time.Second
+	syncChainFastPollingInterval      = 1 * time.Second
```

Comment:
Start this one with syncToChain too, for consistency.

Reply:
of course